src/sql/src/statements/insert.rs

```diff
@@ -39,20 +48,51 @@ impl Insert {
         }
     }
 
-    pub fn values_body(&self) -> Result<Option<Vec<Vec<Value>>>> {
-        let values = match &self.inner {
+    /// Extracts the literal insert statement body if any, and returns `None` if the body of insert statement contains non literal expressions.
```

Comment:
```suggestion
    /// Extracts the literal insert statement body.
```
there's no `None` now